compiler/builtins/string_f64.ts

```diff
@@ -5,4 +5,154 @@ import type {} from './porffor.d.ts';
 export const String = function (value: any): bytestring {
   if (!new.target && Porffor.rawType(value) == Porffor.TYPES.symbol) return __Symbol_prototype_toString(value);
   return __ecma262_ToString(value);
-};
\ No newline at end of file
+};
+
+export const __String_prototype_concat = (_this: string, arg: string) => {
```

Comment:
Why is this in the same PR? Isn't it a separate thing making some new prototype methods (ty!)

Reply:
Ah, I forgot to elaborate on that, I could just edit the previous method if you would like me to do that. But, the previous function is incorrect with my changes, mostly because it copies the entire page instead of just the length of the string.